pkg/kapp/logs/pod_container_log.go

```diff
@@ -44,6 +44,30 @@ func NewPodContainerLog(
 }
 
 func (l PodContainerLog) Tail(ui ui.UI, cancelCh chan struct{}) error {
+	for {
+		linePrefix := ""
+		if len(l.opts.LinePrefix) > 0 {
+			linePrefix = l.opts.LinePrefix + " | "
+		}
```

Comment:
I think we can keep this outside of the for loop.

Reply:
Adopted